src/components/Line.tsx

```diff
@@ -8,12 +8,19 @@ import { renderTextTrainlabel } from '../labels';
 import { Train } from './Train';
 import { PopoverContainerContext, getTrainRoutePairsForLine, setCssVariable } from './util';
 import { getInitialDataByKey } from '../initialData';
-import { Line as TLine, Pair, StationPositions } from '../types';
+import { Line as TLine, Pair, StationPositions, VehiclesAge } from '../types';
 import { MBTAApi } from '../useMbtaApi';
 
+const AGE_WORD_MAP = new Map<VehiclesAge, string>([
+    ['new_vehicles', ' new '],
+    ['old_vehicles', ' old '],
+    ['vehicles', ' '],
+]);
+
 interface LineProps {
     api: MBTAApi;
     line: TLine;
+    age: string;
```

Comment:
I think this can be of type `VehicleAge`, right?

Reply:
```suggestion
    age: VehiclesAge;
```